esp-hal/src/gpio/rtc_io.rs

```diff
@@ -14,7 +14,7 @@
 //! chip from Deep-sleep.
 //!
 //! # Example
-//! ```no_run
+//! ```ignore
```

Comment:
```suggestion
//! ```rust,ignore
```

Reply:
I added `rust` to all snippets (hopefully), thanks!